lib/linter-elm-make.js

```diff
@@ -398,6 +413,11 @@ export default {
     });
   },
   doLint(editorFilePath, projectDirectory, editor) {
+    if (this.skipNextLint) {
```

Comment:
This really is the unholiest of hacks... The problem is that the following sequence occurs on save

* save
* lint
* removeUnusedImports
* save
* ~~lint~~

Note the final lint which is triggered by the second save but which is undesired because the linting has already been completed/saved and we don't want to fire removeUnusedImports again.